Makefile

```diff
@@ -175,3 +181,32 @@ $(foreach lang,$(BUILDERS_LANGS),$(eval $(call GEN_PULL_EXTENSION_BUILDER_IMAGE_
 
 .PHONY: builders.pull
 builders.pull: $(foreach lang,$(BUILDERS_LANGS), pull/builder/$(lang))
+
+##@ Code quality and integrity
+
+LINT_OPTS ?= --timeout 5m
+.PHONY: lint
+# generate must be called while generated source is still used
+lint: generate $(GOLANGCI_LINT) $(SHFMT) $(LICENSER) .golangci.yml  ## Run the linters
+	@echo "--- lint ---"
+	@$(SHFMT) -d .
+	@$(LICENSER) verify -r .
+	@$(GOLANGCI_LINT) run $(LINT_OPTS) --config .golangci.yml
```

Comment:
I think we should also check if it's goimport formatted with something like `${goimport execute ...} && git diff --exit 1`

Reply:
ok I will add this to the "check" task in make similar to some other projects

Reply:
PTAL! thanks and good idea

Reply:
super! :+1: